ProjectBasic/app/src/main/java/com/example/moviedb/screen/TabFragmentType.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.moviedb.screen
+
+enum class TabFragmentType(val value: Int) {
+    HOME_FRAGMENT(0),
+    FAVORITE_FRAGMENT(1)
```

Comment:
chú nên tạo 1 package "annotaion" trong package "utils" để chứa các annotation này nhé.
```suggestion
@IntDef(MainPage.HOME_PAGE,
        MainPage.FAVORITE_PAGE)

annotation class MainPage {
    companion object {
        const val HOME_PAGE = 0
        const val FAVORITE_PAGE = 1
    }
}
```

use 
``
when (position) {
            MainPage.HOME_PAGE -> //Todo
            MainPage.FAVORITE_PAGE -> //Todo
        }